AppCreationScripts/Apps.json

```diff
@@ -0,0 +1,42 @@
+{
+  "Sample": {
+    "Title": "Integrating Azure AD into a Python web application",
+    "Level": 400,
+    "Client": "Python, MSAL.Python"
+  },
+  "AppRegistrations": [
+    {
+      "x-ms-id": "PythonWebApp",
+      "x-ms-name": "ms-identity-python-webapp",
+      "x-ms-version": "2.0",
+      "logoutUrl": "http://localhost:5000/logout",
+      "replyUrlsWithType": [
+        {
+          "url": "http://localhost:5000/getAToken",
+          "type": "Web"
+        }
+      ],
+      "x-ms-passwordCredentials": "Auto",
+      "oauth2AllowImplicitFlow": false,
+      "oauth2AllowIdTokenImplicitFlow": false,
+      "requiredResourceAccess": [
+        {
+          "x-ms-resourceAppName": "Microsoft Graph",
+          "resourceAppId": "00000003-0000-0000-c000-000000000000",
+          "resourceAccess": [
+            {
+              "id": "e1fe6dd8-ba31-4d61-89e7-88639da4683d",
+              "type": "Scope",
+              "x-ms-name": "user.read"
+            },
+            {
+              "id": "b340eb25-3456-403f-be2f-af7a0d370277",
+              "type": "Scope",
+              "x-ms-name": "User.ReadBasic.All"
```

Comment:
This is in prevision of the evolutions discussed recently by email.

Reply:
Sounds good. So this `apps.json` is the metadata only used by the Portal? For such kind of file that is not used by this repo itself, ideally we would like to have a comment to document so, to avoid future confusion.
Shall we try add a second line into this file, like this?

```
{
    "comment": "This file will be used by App Registration Portal",
    "Sample": {
        ...
    },
    ...
}
```

Reply:
yes. I'm not sure what the portal will do if we add another attribute in the json file. Probably ok, but we need to check with Shiv.

Reply:
Fair enough. In that sense, @shivbijlani should be the one who reviewed this PR in the first place. Nonetheless, CCing him here. :-)